coil-base/src/main/java/coil/ImageLoaderBuilder.kt

```diff
@@ -33,24 +34,12 @@ class ImageLoaderBuilder(private val context: Context) {
     private var defaults = DefaultRequestOptions()
 
     /**
-     * Set the [OkHttpClient] to be used for network requests.
+     * Set the [Call.Factory] to be used for network requests.
      *
-     * Prefer `okHttpClient(OkHttpClient.Builder.() -> Unit)` if possible,
-     * as the default [OkHttpClient] instance is optimized for Coil.
+     * If using an [OkHttpClient] instance under the hood, use [applyCoilOptimizations] if possible to optimize it for Coil.
      */
-    fun okHttpClient(client: OkHttpClient) = apply {
```

Comment:
Let's keep this (but remove `okHttpClientBuilder`) for discoverability.

Reply:
fbdd36a